.legacy/git-notes-spec-idea.md

```
@@ -95,13 +104,13 @@ All notes are stored under `refs/notes/gitmind/*`.
 | Namespace              | Purpose                              |
 |------------------------|--------------------------------------|
 | `confidence`           | Degree of belief in a link's validity (0.0–1.0)  
```

markdownlint-cli2 (0.17.2)

Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)